static/src/javascripts/projects/common/modules/experiments/tests/contributions-story.js

```diff
@@ -0,0 +1,176 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-story.html',
+    'text!common/views/contributions-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/modules/commercial/commercial-features',
+    'common/utils/cookies',
+    'common/utils/element-inview'
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsStory,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             commercialFeatures,
+             cookies,
+             ElementInview
+) {
+
+    return function () {
+
+        this.id = 'ContributionsStory20160922';
+        this.start = '2016-09-22';
+        this.expiry = '2016-10-03';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test whether telling the story of the guardian through staggered messages over time results in more contributions than always showing the epic message.';
```

Comment:
Not sure what the epic message is, so I don't quite get the context for this description. I assume that this is another test that had run previously, or is running now? Can we change this description to cater for everyone regardless of other tests that are running, or have run.